tensorflow/core/ops/ctc_ops.cc

```diff
@@ -90,15 +92,16 @@ REGISTER_OP("CTCGreedyDecoder")
     });
 
 REGISTER_OP("CTCBeamSearchDecoder")
-    .Input("inputs: float")
+    .Input("inputs: T")
     .Input("sequence_length: int32")
     .Attr("beam_width: int >= 1")
     .Attr("top_paths: int >= 1")
     .Attr("merge_repeated: bool = true")
     .Output("decoded_indices: top_paths * int64")
     .Output("decoded_values: top_paths * int64")
     .Output("decoded_shape: top_paths * int64")
-    .Output("log_probability: float")
+    .Output("log_probability: T")
+    .Attr("T: {float, double}")
```

Comment:
ditto

Reply:
resolved in 51cd5877412f9d67f2ec180c8087ae8d7a059da1